### PR TITLE
Seeded DB 

### PR DESCRIPTION
**Pull request by kimberlyjk**
all that needs to be done after all the npm installs, is to run the server --> npm start and then check your mongo DB to ensure that overviews have been filled.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+  <title>My test page</title>
+</head>
+
+<body>
+  <div id='overview'></div>
+</body>
+<script type="module" src='../dist/overview.js'></script>
+<script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
```
Comment (djian001) [inline public/index.html:12]: Is there an advantage to using a source script tag dependency vs importing/npm installing it?

File: server/index.js [hunk 2]
```diff
@@ -5,4 +5,61 @@ const port = 3003;
...
+      numberOfSites: 5
+    },
+    owner: {
+      name: 'Anne B.',
+      imageUrl: 'https://krita-artists.org/uploads/default/original/2X/c/cb096de3604544196cb63799a02405a0e32420bf.jpeg'
+}};
+res.send(mockData);
+});
+
+app.patch('/overview/:campId/location', async (req, res) => {
```
Comment (djian001) [inline server/index.js:29]: You mentioned that you were changing to GET already.

Comment (kimberlyjk) [reply server/index.js:29]: yes ma'am!

**Review by djian001 — APPROVED** [review]
Very clean code, great job on your breaking up and making all your different schemas!

File: public/index.html (hunk 1 above)
Comment (kimberlyjk) [reply public/index.html:12]: I believe it has to do with saving storage space? but this is a good question that I am not 100% sure what the answer is.

**Pull request merged**